upload.sh

```diff
@@ -1,3 +1,3 @@
 . .env
 
-npx fusionauth theme:upload $THEME_ID -k $API_KEY -i $TMP_DIR -h $FUSIONAUTH_URL
+npx @fusionauth/cli theme:upload $THEME_ID -k $API_KEY -i $TMP_DIR -h $FUSIONAUTH_URL
```

Comment:
Why do we need to do this?  Is this a more proper way of calling the package?

@ColinFrick is this typical in your experience?

Reply:
It is necessary if `npm install` is not executed prior to running `upload.sh`:
![image](https://github.com/FusionAuth/fusionauth-theme-helper/assets/5517677/1e89754b-e96f-4ba5-a36b-4dbc317bc765)

After running `npm i` it works as intended:
![image](https://github.com/FusionAuth/fusionauth-theme-helper/assets/5517677/2fbbe65d-9b29-46f8-ac8d-370f8556b885)

Reply:
Okay. In that case, @rideam , lets make sure the user has run `npm install` and keep the scripts as they are `npx fusionauth`, please.

Reply:
Cool, `npm install` is already mentioned earlier in the installation section of the README